app/index.js

```diff
@@ -217,27 +221,27 @@ module.exports = yeoman.generators.Base.extend({
       this.log();
     }
 
-    this.log('Next steps:');
+    g.log('Next steps:');
```

Comment:
Same here.  `this.log` should not be globalized.  `this.log(g.f('Next steps:'));`